src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -397,6 +618,79 @@ function isCtrlKey(e) {
 .sf-dump-str-expand .sf-dump-str-expand {
     display: none;
 }
+.sf-dump-public.sf-dump-highlight,
+.sf-dump-protected.sf-dump-highlight,
+.sf-dump-private.sf-dump-highlight,
+.sf-dump-str.sf-dump-highlight,
+.sf-dump-key.sf-dump-highlight {
+    background: rgba(111, 172, 204, 0.3);
+    border: 1px solid #7DA0B1;
+    border-radius: 3px;
+}
+.sf-dump-public.sf-dump-highlight-active,
+.sf-dump-protected.sf-dump-highlight-active,
+.sf-dump-private.sf-dump-highlight-active,
+.sf-dump-str.sf-dump-highlight-active,
+.sf-dump-key.sf-dump-highlight-active {
+    background: rgba(253, 175, 0, 0.4);
+    border: 1px solid #ffa500;
+    border-radius: 3px;
+}
+.sf-dump-search-hidden {
+    display: none;
+}
+.sf-dump-search-wrapper {
+    float: right;
```

Comment:
is there a clearfix behavior in the styles of the parent block ? If no, it does *not* ensure the min height